### PR TITLE
Subjects missing persons

### PR DESCRIPTION
**Pull request by olegkolyaka78**
For FBI search I added criteria for selecting missing people whose Subjects include words Missing Persons, it will narrow the search.

### CODE REVIEW

File: 1 [hunk 1]
```diff
@@ -0,0 +1,35 @@
+[33mcommit de00ad8581ec7c0b0ae7f768b9e5c4d51af3f7aa[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```
Comment (mechamanny) [inline 1]: I think this file should not be included in the PR, not sure what created it!

File: app/javascript/packs/ajax_operations.js [hunk 2]
```diff
@@ -602,31 +601,38 @@ function handle_ajax(event)
...
-            }
+            for (let s in fbiApiResponseData.items[x].subjects)
+              if (fbiApiResponseData.items[x].subjects[s] === 'Kidnappings and Missing Persons' ||
+                  fbiApiResponseData.items[x].subjects[s] === 'ViCAP Missing Persons' )
+              {
+                if (fbiEntry === 'all')
+                {
+
+                     
+                          text += "<tr>";
```
Comment (mechamanny) [inline app/javascript/packs/ajax_operations.js:612]: I know you didn't create this way of handling the data, but this seems very brittle. At the very least this should be extracted to a separate function.

**Review by inlibra-cpa — APPROVED** [review]
I'm approving, though, I do not have too much of an expertise on the issue.

**Review by inlibra-cpa — APPROVED** [review]
Considering my experience, I don't see contradictions.

**Comment by jimenezmiguela** [comment]
Hi Oleg, what are the conflicts?

**Comment by olegkolyaka78** [comment]
> Hi Oleg, what are the conflicts?

The first conflict is presence of nokogiri (1.13.1-x86_64-linux) in Gemfile in case of using vagrant.

The second conflict relates to adding condition in searching data:

for (let s in fbiApiResponseData.items[x].subjects)
              if (fbiApiResponseData.items[x].subjects[s] === 'Kidnappings and Missing Persons' ||
                  fbiApiResponseData.items[x].subjects[s] === 'ViCAP Missing Persons' )

**Comment by olegkolyaka78** [comment]
Created new pull request to avoid conflicts
